docs/04_web/05_micro-front-ends/01_introduction.mdx

```diff
@@ -11,10 +11,6 @@ tags:
   - introduction
 ---
 
-Micro-front-end architecture is a design approach in which a front-end app is decomposed into individual,
-semi-independent __micro applications__ working loosely together. The micro-front-end concept is vaguely inspired by, and named
-after, microservices. Independent development teams can collaborate on a front-end app more easily using this approach,
-and can each have their own release cadence. At Genesis, we use a number of technologies to enable this functionality,
-including [Webpack 5 Module Federation](https://webpack.js.org/concepts/module-federation/).
+A micro front-end is a collection of components that meet common requirements. For example, [entity management](/web/micro-front-ends/foundation-entity-management/) gives you a display grid and a form that can be used to insert and edit. 
 
-We've built out a number of re-usable micro-front-ends that can be used by Genesis-powered applications.
\ No newline at end of file
+So you have ready-made components for common requirements. And yes, a micro front-end can be treated just like a single component. You just declare it in the **-style.ts** file for your page and include the attributes you want.
```

Comment:
suggestion:

for this page I'd keep some of the points

+ include a mention that it's "just like a simple web component too" and maybe add a few of our official MFs (and links to each one)